Tasks/NuGetV0/task.loc.json

```diff
@@ -75,4 +75,4 @@
     "NGCommon_UnableToFindTool": "ms-resource:loc.messages.NGCommon_UnableToFindTool",
     "Warning_UpdatingNuGetVersion": "ms-resource:loc.messages.Warning_UpdatingNuGetVersion"
   }
-}
+}
```

Comment:
nit: newline

Reply:
This is generated when i build the task, what does nit mean?

Reply:
It's short for nitpick. It means minor recommendation

Reply:
I see, yeah i didnt change anything in the original task.json and this change is auto generated, not sure if it's because one of the older commits did not check the correct changes in.